src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -111,6 +113,34 @@ function WorkspaceMoreFeaturesPage({policy, route}: WorkspaceMoreFeaturesPagePro
                 setIsDisableExpensifyCardWarningModalOpen(true);
             },
         });
+        spendItems.splice(2, 0, {
```

Comment:
Why are we doing splice (and in the existing code above)? Why not do `push` instead?

Reply:
we need a splice for both this one and one on top to place Expensify card and Company cards in the middle of existing spend items. This code should be removed when both features will be ready for release
we already have TODO above:
"// TODO remove this when feature will be fully done, and move spend item inside spendItems array"

Reply:
What do you mean in the middle?
- Line 81 creates the `spendItems` array with 1 item
- Line 99 adds one item in position 1, which is right after the one in the previous step
- This line adds one item in position 2, which is right after the one in the previous step

Reply:
Oh yeah good point